src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace Backend\Modules\Location\Domain;
+
+use Common\Locale;
+use Backend\Core\Language\Locale as BackendLocale;
+use Frontend\Core\Language\Locale as FrontendLocale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="location")
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class Location
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var Locale
+     *
+     * @ORM\Column(type="locale", name="language")
+     */
+    private $locale;
+
+    /**
+     * @TODO: map this to some sort of Extra entity
+     * @var integer
+     *
+     * @ORM\Column(type="integer", name="extra_id")
```

Comment:
Reden dat je column-name specifiek meegeeft?

Reply:
It's to be backwards compatible but it's not completely necessary (there are database changes anyway) so I might remove these

Reply:
@StijnVrolijk I disagree, you can do this perfectly in a BC way

Reply:
@carakas you're going to have the doctrine comments either way so unless that doesn't break BC it can't be avoided

Reply:
that doesn't break BC. If they are not there, doctrine will just keep saying that you need to update your schema to add the comments. It can work perfectly without them